[JENKINS-56398] Mobile friendly login / load / restart screens

zbynek opened the pull request:
See [JENKINS-56398](https://issues.jenkins-ci.org/browse/JENKINS-56398).

### Proposed changelog entries

* Mobile friendly layout of the login, loading and restart screens

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

Note: the padding change makes no difference for login, but looks slightly better for restart. 

## Login before
![login-current](https://user-images.githubusercontent.com/1105305/53830777-7ce29300-3f83-11e9-9aa9-0e55a5f1ff70.png)
## Login after
![login-mobile-friendly](https://user-images.githubusercontent.com/1105305/53830794-83710a80-3f83-11e9-9d8c-fd915be03fed.png)
## Restart before
![restart-current](https://user-images.githubusercontent.com/1105305/53830813-8d930900-3f83-11e9-98a3-e1308b77e5e8.png)
## Restart after
![restart-mobile-friendly](https://user-images.githubusercontent.com/1105305/53830819-92f05380-3f83-11e9-85ec-e44a64eed1c6.png)

zbynek committed:
JENKINS-56398 Mobile friendly login / load / restart screens

alecharp commented:
Thank you for your contribution. Could you please include a pre and post fix screenshot on a desktop/laptop screen size? This way, anyone can see the difference and the impact? Thank you.

alecharp reviewed (approved):
LGTM, with screenshot it will be perfect.

oleg-nenashev reviewed (approved):
Should be ok from what I know about Web UIs and CSS (spoiler: not so much)

zbynek committed:
[JENKINS-56398] increase line-height of heading

zbynek commented:
I updated the original comment with screenshots. Looking at the `restart-mobile-friendly` screenshot again I realized the heading lines were too close to each other. Fixed in second commit, result is below.
![restart-mobile-friendly-2](https://user-images.githubusercontent.com/1105305/53835106-f29f2c80-3f8c-11e9-885f-8e8dc24be790.png)

@alecharp thanks for reviewing this! You also mentioned desktop screenshots: for desktop browsers this pull makes no difference. All screenshots were taken in Chrome dev tools emulating Pixel 2, I also tried the login on a physical Android 5'' phone.

scherler reviewed (approved):
lgtm

halkeye commented:
> for desktop browsers this pull makes no difference

There's a padding change, what does that actually do?

> [Second Screenshot]

I was going to bring up the squished header, so with that fixed, I'm :+1: myself

daniel-beck commented:
How does this impact the signup form when having user signup enabled in the Jenkins user database?

zbynek committed:
[JENKINS-56398] mobile friendly layout for signup form

zbynek commented:
@daniel-beck thanks, good point. Viewport added for signup in the last commit
![signup-mobile-friendly](https://user-images.githubusercontent.com/1105305/53919772-4f711480-406b-11e9-95a4-e36f1e9d2bad.png)

@halkeye the padding was meant to make sure the text on reload screen does not go too close to vertical borders, but it also affects the current version of signup, so I removed the padding again. Restart is now like this:
![restart-no-padding](https://user-images.githubusercontent.com/1105305/53920249-c65add00-406c-11e9-9d35-c67308ac5189.png)
I hope the latest version is OK.

oleg-nenashev commented:
I think this is ready to go.
Thanks @zbynek !
